code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -319,6 +319,10 @@
 		H.reagents.remove_reagent(chem.type, REAGENTS_METABOLISM)
 		return 1
 
+/datum/species/golem/wood/holy //slightly upgraded wood golem, for the plant sect
+	id = "holy wood golem"
+	speedmod = 1 //wood golems aren't very good, so the holy ones are slightly faster so that you don't put in a bunch of hardwork to downgrade yourself
+
```

Comment:
Add "changesource_flags = MIRROR_BADMIN" and "random_eligible = FALSE" so it's the same as other church golems, only available via the church conversion.